src/state/tooltipSlice.ts

```diff
@@ -131,9 +131,16 @@ export type TooltipState = {
    * may render multiple tooltip payloads.
    */
   tooltipItemPayloads: ReadonlyArray<TooltipPayloadConfiguration>;
+  // whether or not the Tooltip is active (from Redux' perspective)
+  active: boolean;
```

Comment:
I am afraid this information (and all the other activeX) needs to be more granular. Tooltip can be activated based on clicks or hovers, and the trigger is defined on the Tooltip element. Axis and item "tooltipEventType" also have different activation rules.

And if you look closely, there is already activeIndex and `activeClick` + `activeHover`, four of them, in the objects above: 

```
  axisInteraction: {
    activeClick: boolean;
    activeHover: boolean;
```

and there's a Redux selector selecting that: `selectIsTooltipActive`. Can we use that instead of adding a new property?

Reply:
If we have this information in state split down for interactions, we unlock two things:

1) Graphical elements can send events without knowing what is the Tooltip trigger, and
2) ability to have multiple tooltips with different behaviour.

Reply:
lol my eyes literally glazed over all of the existing state in this slice 😅

Reply:
> ability to have multiple tooltips with different behaviour

At the moment though we could only have two though yeah? One with click behavior one with hover behavior? Unless we have a tooltipId or something

Reply:
In master branch, the renderMap has a "once" property which makes it lookup and render the first Tooltip and ignore others. So, one max.

Reply:
I mean in the current state of 3.x. With `activeClick...` and `activeMouseOver...` we could only have one of each of those two types of Tooltips. Else we would need to store more info

Reply:
Ah right. There can be as many as we want. Maybe one for each coordinate. Or one tooltip for each tick on a Line idk.